.github/workflows/release.yaml

```diff
@@ -68,35 +124,16 @@ jobs:
             ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/lib/libz.a \
             ${GITHUB_WORKSPACE}/build/libgit2-darwin-arm64/lib/libz.a 
 
-          tar -zcvf darwin-libs.tar.gz ./libgit2-darwin
+          tar -zcvf darwin-libs.tar.gz libgit2-darwin
         env:
           MACOSX_DEPLOYMENT_TARGET: 10.15
-
-      - name: Prepare
-        id: prep
-        run: |
-          VERSION="${{ github.event.inputs.tag }}-${GITHUB_SHA::8}"
-          if [[ $GITHUB_REF == refs/tags/* ]]; then
-            VERSION=${GITHUB_REF/refs\/tags\//}
-          fi
-          echo ::set-output name=VERSION::${VERSION}
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        with:
-          release_name: ${{ steps.prep.outputs.VERSION }}
-          tag_name: ${{ steps.prep.outputs.VERSION }}
-          draft: false
-          prerelease: true
       - name: Upload Release Asset
         id: upload-release-asset 
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ github.token }}
         with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          upload_url: ${{ needs.github_release.outputs.release_upload_url }}
```

Comment:
Is this a replace typo?

Reply:
No, the release is now used in more than one place, so the release creation is now happening at the `github_release` job, which has the output `release_upload_url`.

This is the same as we had merged for `libgit2-1.1.1` branch last week.

Reply:
Yep, it is under `steps` within the same job. But if it comes from a workflow you depend on you should use `needs` instead:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idoutputs

Reply:
Cool, I didn't know that, thanks!